### DISCUSSION TITLE
Finding the best markdown editor

### DISCUSSION THREAD

**Discussion by losolio** [losol/eventuras#736] [General]
For now we are using an outdated build of CKEditor5 in the email all in the admin/eventapge. Possible options are: 

- [react-mde](https://www.npmjs.com/package/react-mde):
- [mantine/tiptap](https://mantine.dev/others/tiptap/) - no markdown support
- [ckeditor5](https://ckeditor.com/docs/ckeditor5/latest/features/markdown.html): This is a powerful framework with modular architecture that provides every type of WYSIWYG editing solution imaginable. It has an LGPL license. Only internal dependencies. There is some trouble using it in Next js.

**Comment by losolio** [comment]
This might also be some options: 

* https://tiptap.dev/ - headless editor, but [no markdown support from the core edit according to docs](https://tiptap.dev/guide/output#not-an-option-markdown).
* https://lexical.dev/ facebooks react editor

**Comment by losolio** [comment]
This is a really cool editor, looks like Notion: 
* [Editor.js](https://editorjs.io/) - outputs json, needs to find a markdown plugin to use

**Comment by losolio** [comment]
Good old one, but no markdown support: 
* [TinyMCE](https://www.tiny.cloud/) - [considering markdown support](https://www.tiny.cloud/roadmap/), but no support yet.

**Comment by meinte** [comment]
https://github.com/uiwjs/react-md-editor looks popular with active development
